thundra/plugins/log/log_plugin.py

```diff
@@ -10,30 +13,73 @@ def __init__(self):
             'after:invocation': self.after_invocation
         }
         self.log_data = {}
+        self.tracer = ThundraTracer.getInstance()
+        self.scope = None
 
     def before_invocation(self, data):
         context = data['context']
         logs.clear()
+        function_name = getattr(context, constants.CONTEXT_FUNCTION_NAME, None)
+        active_span = self.tracer.get_active_span()
         self.log_data = {
-            'transactionId': data['transactionId'],
-            'applicationName': getattr(context, 'function_name', None),
+            'type': "Log",
+            'agentVersion': '',
+            'dataModelVersion': constants.DATA_FORMAT_VERSION,
             'applicationId': utils.get_application_id(context),
+            'applicationName': function_name,
             'applicationVersion': getattr(context, constants.CONTEXT_FUNCTION_VERSION, None),
-            'applicationProfile': utils.get_environment_variable(constants.THUNDRA_APPLICATION_PROFILE, ''),
-            'applicationType': 'python'
+            'applicationStage': '',
```

Comment:
should be taken from environment variable `thundra_agent_lambda_application_stage`